app/views/notification_mailer/release_email.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="emailbkgd dark-scheme">
+  <div class="top_section dark-scheme">
+    <%= image_tag attachments['markus_logo_big.png'].url, :class => "logo_header" %>
+  </div>
+  <hr class="divider">
+  <p class="to_text dark-scheme">
+    <%= t('greeting', first_name: @user.first_name, last_name: @user.last_name) %>
+  </p>
+  <p class="body_text dark-scheme">
+    <%= t('text_body', assignment: @grouping.assignment.short_identifier) %>
+  </p>
+  <div class="button_enclosure">
+    <%= link_to t('button_text'),
+                view_marks_assignment_submission_result_url(assignment_id: @grouping.assignment.id,
+                                                            submission_id: @grouping.current_submission_used.id,
+                                                            id: @grouping.current_result.id),
+                class: 'redirectbutton' %>
+  </div>
+  <p class="alert_text dark-scheme">
+    <%= t('disclaimer') %>
+  </p>
+</div>
+
```

Comment:
Delete the extra blank lines at the end of this file (you should end with only one blank line).